pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
+	if err != nil {
+		log.Fatalf("Failed to load compose files: %v", err)
+	}
+
+	for i, file := range files {
+		// ensure main docker-compose is first
```

Comment:
I'd rather use some kind of more elegant sorting technique to put docker-compose first and override last, for example... but I don't know if those are our names or not. But explicit ordering (perhaps a function whose job is to order the files instead of inline code) would make this more robust for the future as we discover new needs.